resources/qml/Menus/ConfigurationMenu/ConfigurationItem.qml

```diff
@@ -14,6 +14,23 @@ Button
     property var configuration: null
     hoverEnabled: true
 
+    property bool isValidMaterial:
+    {
+        var extruderConfigurations = configuration.extruderConfigurations
+
+        for (var index = 0; index < extruderConfigurations.length; index++)
+        {
+            var name = extruderConfigurations[index].material ? extruderConfigurations[index].material.name : ""
+
+            if (name == "" || name == "Unknown")
+            {
+                hoverEnabled = false
```

Comment:
The hoverEnabled property shouldn't be set like that. It's better to just use `hoverEnabled: isValidMaterial `

Reply:
it is set to true (Line: 15,  hoverEnabled: true), check above code.

Reply:
Yes, but the problem is that this breaks the binding. If it becomes a valid material during runtime, then it would still not have a hover.

Currently that is no problem because we can't get materials that are already on the printer during runtime (you have to restart Cura after you install a package), but if we get that working better then this would be a bug.